app/views/groups/manage/clients/existing_clients.scala.html

```diff
@@ -43,18 +43,37 @@
     paginationMetaData: Option[PaginationMetaData] = Some(PaginationMetaData(true, true, 0, 1, 10, 1, 10))
 )(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
 
+@showRemoveLink = @{!groupClients.isEmpty && groupClients.length > 1}
```

Comment:
groupClients could be empty due to a search? This should be based off the GroupSummary client count

Reply:
ok. see now